README.rst

```diff
@@ -5,6 +5,9 @@ Zephyr Project
 
    <a href="https://bestpractices.coreinfrastructure.org/projects/74"><img
    src="https://bestpractices.coreinfrastructure.org/projects/74/badge"></a>
+   <img
```

Comment:
Is there a link you can add (as there is for the bestpractices badge) that folks can click on for more details?

Reply:
maybe this?  https://app.shippable.com/github/zephyrproject-rtos/zephyr/dashboard

Reply:
this will be just confusing, I am planning to have something more specific for this, but would start with the badge without a link.